Sources/xcbeautify/root.swift

```diff
@@ -14,68 +14,56 @@ private func configuration(command: Command) {
               value: false,
               description: "Only print tasks that have warnings or errors.",
               inheritable: true)
-        ])
+    ])
     
     command.add(flags: [
         .init(longName: "quieter",
               value: false,
               description: "Only print tasks that have errors.",
               inheritable: true)
-        ])
-
+    ])
+    
+    command.add(flags: [
+        .init(longName: "isCI",
+              value: false,
+              description: "Print test result too under quiet/quiter flag")
+    ])
+    
     command.add(flags: [
         .init(shortName: "v",
               longName: "version",
               value: false,
               description: "Prints the version",
               inheritable: true)
-        ])
-
+    ])
+    
```

Comment:
Remove whilespace-only lines (some places below too). (There's a setting in Xcode that automatically trims these for you)

```suggestion
```